pkg/sql/sem/tree/show.go

```diff
@@ -599,15 +599,6 @@ func (node *ShowSyntax) Format(ctx *FmtCtx) {
 	}
 }
 
-// ShowTransactionStatus represents a SHOW TRANSACTION STATUS statement.
-type ShowTransactionStatus struct {
```

Comment:
we still need `ShowTransactionStatus` as a special case, since it's an observer statement (i.e. doesn't use the planner at all)